cmd/pj-rehearse/main.go

```diff
@@ -281,21 +283,22 @@ func rehearseMain() int {
 	metrics.RecordOpportunity(toRehearseClusterProfiles, "cluster-profile-change")
 	toRehearse.AddAll(toRehearseClusterProfiles)
 
-	rehearsals := rehearse.ConfigureRehearsalJobs(toRehearse, prConfig.CiOperator, prNumber, loggers, o.allowVolumes, changedTemplates, changedClusterProfiles)
-	metrics.RecordActual(rehearsals)
-	if len(rehearsals) == 0 {
+	jobConfigurer := rehearse.NewJobConfigurer(toRehearse, changedPeriodics, prConfig.CiOperator, prNumber, loggers, o.allowVolumes, changedTemplates, changedClusterProfiles)
+	presubmits, periodics := jobConfigurer.ConfigureRehearsalJobs()
+	metrics.RecordActual(presubmits)
+	if len(presubmits) == 0 && len(changedPeriodics) == 0 {
 		logger.Info("no jobs to rehearse have been found")
 		return 0
-	} else if len(rehearsals) > o.rehearsalLimit {
+	} else if len(presubmits) > o.rehearsalLimit {
 		jobCountFields := logrus.Fields{
 			"rehearsal-threshold": o.rehearsalLimit,
-			"rehearsal-jobs":      len(rehearsals),
+			"rehearsal-jobs":      len(presubmits) + len(changedPeriodics),
```

Comment:
`changedPeriodics` does not seem right - those are all that were changed, but we do not necessarily rehearse them all. Should be `len(periodics)`?